Error running medium tests

When I try to run the go medium tests I get the following error:
```bash
[nose.plugins.manager] DEBUG: DefaultPluginManager load plugin nose_json = nose_json.plugin:JsonReportPlugin
[nose.plugins.manager] DEBUG: DefaultPluginManager load plugin cov = nose_cov:Cov
[nose_cov] DEBUG: nose-cov options
[nose_cov] DEBUG: nose-cov configure
F
======================================================================
FAIL: Install Go from scratch test case
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jravetch/Projects/ubuntu-make/tests/large/test_go.py", line 72, in test_default_go_install
    self.assertEqual(output, "hello, world")
AssertionError: 'bash: warning: setlocale: LC_ALL: cannot [244 chars]orld' != 'hello, world'
- bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)hello, world
+ hello, world


----------------------------------------------------------------------
Ran 1 test in 8.670s

FAILED (failures=1)

```
When I run the rust medium tests, I get a similar error:
```bash
[nose.plugins.manager] DEBUG: DefaultPluginManager load plugin nose_json = nose_json.plugin:JsonReportPlugin
[nose.plugins.manager] DEBUG: DefaultPluginManager load plugin cov = nose_cov:Cov
[nose_cov] DEBUG: nose-cov options
[nose_cov] DEBUG: nose-cov configure
F..
======================================================================
FAIL: Install Rust from scratch test case
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jravetch/Projects/ubuntu-make/tests/large/test_rust.py", line 77, in test_default_rust_install
    subprocess.check_output(self.command_as_list(cmd_list)).decode("utf-8").strip().split(":"))
AssertionError: '/home/user/.local/share/umake/rust/rust-lang/rustc/lib' not found in ['bash', ' warning', ' setlocale', ' LC_ALL', ' cannot change locale (en_US.UTF-8)\r\n/bin/bash', ' warning', ' setlocale', ' LC_ALL', ' cannot change locale (en_US.UTF-8)\r\nbash', ' warning', ' setlocale', ' LC_ALL', ' cannot change locale (en_US.UTF-8)\r\n/home/user/.local/share/umake/rust/rust-lang/rustc/lib']

----------------------------------------------------------------------
Ran 3 tests in 24.831s

FAILED (failures=1)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -10,6 +10,10 @@ MAINTAINER	Didier Roche <didrocks@ubuntu.com>
 # Set the env variable DEBIAN_FRONTEND to noninteractive
 ENV DEBIAN_FRONTEND noninteractive
 
+#Set default locale
+ENV LANG C.UTF-8
+RUN locale-gen en_US.UTF-8
+
 ADD debian/control /tmp/
 ADD docker/umake_docker.pub /tmp/
 ADD tests/data/developer.android.com.crt /usr/local/share/ca-certificates/

